earth_enterprise/src/portableserver/build.py

```diff
@@ -0,0 +1,256 @@
+#! /usr/bin/python
+#-*- Python -*-
+#
+# Copyright 2017 GEE Open Source Team <github.com/google/earthenterprise>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Builds Portable server for Linux and Windows.
+"""
+
+import datetime
+import distutils.dir_util
+import os
+import os.path
+import shutil
+import sys
+
+SELF_PATH = os.path.realpath(__file__)
```

Comment:
It seems to me that the longer a line becomes, the less readable it is. I prefer having it in multiple parts, but having it on one line is readable enough for me, as well.